iso_templates/initrd_init_template

```diff
@@ -104,6 +104,31 @@ mount_root() {
     fi
 }
 
+wait_for_keystrokes() {
+    while [ true ] ; do
+        read -t 2 -n 1
+        if [ "$?" = 0 ] ; then
+            break
+        fi
+    done
+}
+
+check_iso_verify() {
+    local ret="$1"
+    local msg="$2"
+    [ "$ret" -ne 0 ] && { echo_tty_kmsg "[${FAILURE} FAIL ${NORMAL}] $msg: Press any key to reboot your system"; wait_for_keystrokes; shell_trap "$msg"; }
```

Comment:
We don't want Press a key message on failures, only for success before continuing the boot.

```suggestion
    [ "$ret" -ne 0 ] && { echo_tty_kmsg "[${FAILURE} FAIL ${NORMAL}] $msg; shell_trap "$msg"; }
```

Reply:
Done